integrations/zoom/checks.go

```diff
@@ -46,13 +53,74 @@ func test(v sdkservices.Vars) sdkintegrations.OptFn {
 		case "":
 			return sdktypes.NewStatus(sdktypes.StatusCodeWarning, "Init required"), nil
 		case integrations.OAuthDefault, integrations.OAuthPrivate:
-			// TODO(INT-247): Implement.
+			return oauthTest(ctx, v, o, vs)
 		case integrations.ServerToServer:
-			// TODO(INT-247): Implement.
+			return ServerToServerTest(ctx, vs)
 		default:
 			return sdktypes.NewStatus(sdktypes.StatusCodeError, "Bad auth type"), nil
 		}
-
-		return sdktypes.NewStatus(sdktypes.StatusCodeOK, ""), nil
 	})
 }
+
+// oauthTest verifies the OAuth authentication with Zoom API using the "me" context.
+// (Based on: https://developers.zoom.us/docs/integrations/oauth/#the-me-context).
+func oauthTest(ctx context.Context, v sdkservices.Vars, o sdkservices.OAuth, vs sdktypes.Vars) (sdktypes.Status, error) {
+	// TODO(INT-338): Support private OAuth.
+	// Check if the token is expired and refresh it.
+	desc := common.Descriptor("zoom", "", "")
```

Comment:
No need to redefine `desc` here, this file already has access to the `desc` global var in `integration.go`